invenio_vocabularies/contrib/awards/schema.py

```diff
@@ -61,6 +61,12 @@ class AwardSchema(BaseVocabularySchema, ModePIDFieldVocabularyMixin):
 
     organizations = fields.List(fields.Nested(AwardOrganizationRelationSchema))
 
+    description = SanitizedUnicode()
+
+    start_date = fields.String()
```

Comment:
minor: this might be better to be `ISODateString` from marshmallow utils so that we do some validation on the string before indexing to search (which would fail for invalid ISO Date strings)